sdk/src/bigint.rs

```diff
@@ -21,12 +21,12 @@ impl TryFrom<EnvVal<RawVal>> for BigInt {
 impl TryFrom<EnvObj> for BigInt {
     type Error = ();
 
-    fn try_from(obj: EnvObj) -> Result<Self, Self::Error> {
-        if obj.as_tagged().is_obj_type(ScObjectType::Bigint) {
+    fn try_from(_obj: EnvObj) -> Result<Self, Self::Error> {
```

Comment:
If you merge the main branch into this PR this should be unnecessary.

Reply:
Fixed.